app/colors/catppuccin.css

```diff
@@ -0,0 +1,200 @@
+[current-mode=search] #mode-container::before {content: url("data:image/svg+xml;charset=UTF-8,%3csvg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg' fill='currentColor'%3e%3cpath d='M15.25 0a8.25 8.25 0 0 0-6.18 13.72L1 22.88l1.12 1 8.05-9.12A8.251 8.251 0 1 0 15.25.01V0zm0 15a6.75 6.75 0 1 1 0-13.5 6.75 6.75 0 0 1 0 13.5z'/%3e%3c/svg%3e");}
+[current-mode=pointer] #mode-container::before {content: url("data:image/svg+xml;charset=UTF-8,%3csvg xmlns='http://www.w3.org/2000/svg' id='mdi-cursor-default-outline' viewBox='0 0 24 24'%3e%3cpath d='M10.07,14.27C10.57,14.03 11.16,14.25 11.4,14.75L13.7,19.74L15.5,18.89L13.19,13.91C12.95,13.41 13.17,12.81 13.67,12.58L13.95,12.5L16.25,12.05L8,5.12V15.9L9.82,14.43L10.07,14.27M13.64,21.97C13.14,22.21 12.54,22 12.31,21.5L10.13,16.76L7.62,18.78C7.45,18.92 7.24,19 7,19A1,1 0 0,1 6,18V3A1,1 0 0,1 7,2C7.24,2 7.47,2.09 7.64,2.23L7.65,2.22L19.14,11.86C19.57,12.22 19.62,12.85 19.27,13.27C19.12,13.45 18.91,13.57 18.7,13.61L15.54,14.23L17.74,18.96C18,19.46 17.76,20.05 17.26,20.28L13.64,21.97Z' /%3e%3c/svg%3e");}
+[current-mode=visual] #mode-container::before {content: url("data:image/svg+xml;charset=UTF-8,%3csvg xmlns='http://www.w3.org/2000/svg' id='mdi-form-textbox' viewBox='0 0 24 24'%3e%3cpath d='M17,7H22V17H17V19A1,1 0 0,0 18,20H20V22H17.5C16.95,22 16,21.55 16,21C16,21.55 15.05,22 14.5,22H12V20H14A1,1 0 0,0 15,19V5A1,1 0 0,0 14,4H12V2H14.5C15.05,2 16,2.45 16,3C16,2.45 16.95,2 17.5,2H20V4H18A1,1 0 0,0 17,5V7M2,7H13V9H4V15H13V17H2V7M20,15V9H17V15H20Z' /%3e%3c/svg%3e");}
+/* tabs */
+#tabs > span {padding: 0.25em;font-size: 0.9em;color: var(--tab-foreground);background: var(--tab-background);display: flex;flex: 1;max-width:20em;overflow: hidden;white-space: nowrap;height: 2.5em; border-top: solid .1em transparent;}
+#tabs > ::before {counter-increment: tab-counter 1;content: counter(tab-counter); font-size: 0.85em;}
+#tabs > span > img {margin: auto 0 auto .25em;padding: 0; padding-right: 0.5em;height: 1em;width: 1em;min-height: 1em;min-width: 1em;pointer-events: none;}
+#tabs .visible-tab {background: var(--visible-tab); color: var(--fg); border-top: .1em solid #74c7ec;}
+#tabs .pinned {min-width: 3em !important;max-width: 3em;width: 3em; padding: 0.25em;}
+#tabs .pinned > span { display: none; }
+#tabs .pinned > img {padding: 0;margin: auto;height: 1.2em;width: 1.2em;min-height: 1.2em;min-width: 1.2em;}
```

Comment:
A lot of this styling is default too, and part of this styling means the suspend color is not being used, so please check with the default colorscheme to remove as much css that's duplicate as possible.